pkg/image/client.go

```diff
@@ -97,6 +116,21 @@ func (i *Client) CreateOrUpdateImage(ctx context.Context, image *images.NodeImag
 		}
 		return err
 	}
+	return nil
```

Comment:
I was thinking if we could simplify this somehow. Doesn't this achieve basically the same thing? What do you think?

```
err := i.Create(ctx, image)
if err != nil && !errors.IsAlreadyExists(err) {
  return err
}

return nil
```